core/org.wso2.carbon.ui/src/main/resources/web/dialog/js/dialog.js

```diff
@@ -286,6 +286,102 @@ CARBON.showPopupDialog = function(message, title, windowHight, okButton, callbac
     }
 };
 
+/**
+ * Display any info inside a jQuery UI's confirmation widget.
+ * @method showPopupConfirm
+ * @param {String} message to display html/text
+ * @param {String} okButton to change ok button text
+ * @param {String} cancelButton to change cancel button text
+ * @return {Boolean}
+ */
+CARBON.showPopupConfirm = function (htmlMessage, title, windowHeight, okButton, cancelButton, callback, windowWidth) {
+    if (!isHTML(htmlMessage)) {
+        htmlMessage = htmlEncode(htmlMessage);
+    }
+    var strDialog = "<div id='dialog' title='" + title + "'><div id='popupDialog'></div>" + htmlMessage + "</div>";
+    var requiredWidth = 750;
+    if (windowWidth) {
+        requiredWidth = windowWidth;
+    }
+    var func = function () {
+        jQuery("#dcontainer").html(strDialog);
+        if (okButton) {
+            jQuery("#dialog").dialog({
+                close: function () {
+                    jQuery(this).dialog('destroy').remove();
+                    jQuery("#dcontainer").empty();
+                    return false;
+                },
+                buttons: {
+                    "OK": function () {
+                        if (callback && typeof callback == "function")
+                            callback();
+                        jQuery(this).dialog("destroy").remove();
+                        jQuery("#dcontainer").empty();
+                        return false;
+                    },
+                    "Cancel": function () {
+                        jQuery(this).dialog('destroy').remove();
+                        jQuery("#dcontainer").empty();
+                        return false;
+                    },
+                },
+                height: windowHeight,
+                width: requiredWidth,
+                minHeight: windowHeight,
+                minWidth: requiredWidth,
+                modal: true
+            });
+        } else {
+            jQuery("#dialog").dialog({
+                close: function () {
+                    jQuery(this).dialog('destroy').remove();
+                    jQuery("#dcontainer").empty();
```

Comment:
Store in a variable and use. `jQuery("#dcontainer")`

Reply:
fixed with 6c2f3c0